path.go

```diff
@@ -432,14 +435,25 @@ func (f *filterAttrVal) apply(p *pather) {
 // text equal to the specified value.
 type filterTextVal struct {
 	val string
+	all bool
```

Comment:
I'd prefer to have a new filter type `filterText` instead of including this in `filterTextVal`.  That way it's analogous to `filterAttr` and `filterAttrVal`.